test/core/libp2p.spec.js

```diff
@@ -149,8 +146,11 @@ describe('libp2p customization', function () {
               enabled: true
             }
           },
-          EXPERIMENTAL: {
-            pubsub: false
+          pubsub: {
+            enabled: false,
+            emitSelf: true,
+            signMessages: true,
+            strictSigning: true
```

Comment:
Please would you [update the README](https://github.com/ipfs/js-ipfs#optionsexperimental) to reflect the movement of this config option out of `EXPERIMENTAL` and also document these new options? 🙏